ghost/core/core/server/data/migrations/versions/5.83/2024-05-20-16-01-50-add-activitypub-actors-keys-table.js

```diff
@@ -0,0 +1,10 @@
+const {addTable} = require('../../utils');
```

Comment:
Let's skip this table for now - we'll just use the keys from settings for timebeing